docs/user/collision-shape-for-physics.rst

```diff
@@ -51,8 +43,40 @@ Posting the query and polling for its result are done through the same function.
 Finally due to the above properties, the number of query points posted from a particular owner should be kept consistent across frames.
 The helper classes always submit a fixed number of points this frame, so satisfy this criteria.
 
-Gerstner Waves CPU
-------------------
+
+Compute Shape Queries (GPU)
+---------------------------
+
+This is the default and recommended choice for when a GPU is present.
+Query positions are uploaded to a compute shader which then samples the ocean data and returns the
+desired results.
+The result of the query accurately tracks the height of the surface, including all wave components and depth caches and other Crest features.
+
+
+Baked FFT Data (CPU)
+--------------------
+
+In scenarios where a GPU is not present such as for headless servers, a CPU option is available.
+
+To use this feature, select a *Shape FFT* component that is generating the waves in a scene and enable the **Enable Baked Collision**.
+Next configure the following options:
+
+-  **Time Resolution** - Frames per second of baked data. Larger values may help the collision track the surface closely at the cost of more frames and increase baked data size.
+-  **Smallest Wavelength Required** - Smallest wavelength required in collision. To preview the effect of this, disable power sliders in spectrum for smaller values than this number. Smaller values require more resolution and increase baked data size.
+-  **Time Loop Length** - FFT waves will loop with a period of this many seconds. Smaller values decrease data size but can make waves visibly repetitive.
+
+Next click **Bake to asset and assign to current settings** and select a path and filename for the result.
+After the bake completes the current active *Animated Waves Sim Settings* will be configured to use this data.
+
+There are currently a few key limitations of this approach:
+
+-  Only a single set of waves from one *Shape FFT* component is supported. This collision does not support multiple sets of waves.
```

Comment:
perhaps these should be some kind of formatted box to highlight them more, but lmk what you think

Reply:
Good idea. I checked using "important" and it worked well.

Reply:
done